packages/www/components/AssetsTable/CreateAssetDialog/index.tsx

```diff
@@ -221,6 +222,7 @@ const CreateAssetDialog = ({
             <Button
               css={{ display: "flex", ai: "center" }}
```

Comment:
Same here - if we touch styling of a component, we should move it entirely to Tailwind.

Reply:
Yes, but again, we will create all of them from scratch, and once we do all of them should have styling with tailwind